mfec/agent.py

```diff
@@ -41,13 +42,7 @@ def __init__(
     def choose_action(self, observation):
         self.time += 1
 
-        # Preprocess and project observation to state
-        obs_processed = np.mean(observation, axis=2)
-        # obs_processed = imresize(obs_processed, size=self.size)
-        obs_processed = np.array(Image.fromarray(
-            obs_processed).resize(self.size))
-  
-        self.state = np.dot(self.projection, obs_processed.flatten())
```

Comment:
A co się stało z randomProjection? Ona jest przeniesiona do main?

Reply:
ok widzę, to można to nazwać już stanem a nie obserwacją? I chyba to powinno być tylko dla MFEC, a nie dla DQN, więc raczej lepiej pasuje w klasie agenta MFEC ten kod od projekcji